virttest/test_setup.py

```diff
@@ -1379,32 +1379,26 @@ def remove_driver(self, driver=None):
         return True
 
     @error_context.context_aware
-    def modprobe_driver(self, driver=None, option=None):
+    def modprobe_driver(self, driver=None):
         """
         Method to modprobe driver
 
         :param driver: driver name
-        :param option: modprobe options to driver
         :return: True on success, False on failure
         """
-        msg = "Loading the driver '%s' without option" % driver
-        error_context.context(msg, logging.info)
         if not driver:
             driver = self.driver
+        msg = "Loading the driver '%s' without option" % driver
+        error_context.context(msg, logging.info)
         cmd = "modprobe %s" % driver
-        if not option:
-            option = self.driver_option
-        if driver and driver != "mlx5_core" and ARCH != 'x86_64':
-            if option:
-                cmd += " %s" % option
         if process.system(cmd, ignore_status=True, shell=True):
             logging.debug("Failed to modprobe driver: %s", driver)
             return False
         if ARCH == 'ppc64le' and self.driver == 'mlx5_core':
             pf_devices = self.get_pf_mlx()
             if (self.get_vfs_count() > 0):
                 for PF in pf_devices:
-                    if not self.set_vf(PF, option):
+                    if not self.set_vf(PF, self.driver_option):
```

Comment:
We can not  `set_vf ` with non-zero value (such as `self.driver_option` here) if  `get_vfs_count `  is not zero.

Reply:
I agree. How about we remove L1403~1408 directly? IMO, it should not be here in the modprobe_driver().  set_vf will be executed later in L1477 anyway.

Reply:
Yes, it will be executed later, agree to move it here.